src/BluetoothListPage.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file BluetoothListPage.js
+ *
+ * @brief Page to bluetooth List 
+ *
+ * @author Lucas Guedes <lucas.guedes@thunderatz.org>
+ * @author Vanderson Santos <vanderson.santos@thunderatz.org>
+ *
+ * @date 06/2021
+ */
+
+import React,{useEffect,useState} from 'react';
+import { SafeAreaView } from 'react-native';
```

Comment:
```suggestion
import {SafeAreaView} from 'react-native';
```

Para ficar igual ao outros :v